# Generated for NaoQi method ALFrameManager.gotoAndStopString
# Goes to a certain frame and pause

# The id of the box containing the box.
string id
# The behavior frame name we want the timeline to go to. If will jump to the starting index of the name given.
string frame
---


